src/widgets/medications/medications-overview.component.tsx

```diff
@@ -198,6 +171,128 @@ export default function MedicationsOverview(props: MedicationsOverviewProps) {
       );
     });
   }
+
+  return (
+    <>
+      {patientMedications && patientMedications.length > 0 ? (
+        <SummaryCard
+          name={t("Active Medications", "Active Medications")}
+          styles={{ width: "100%" }}
+          link={`${props.basePath}`}
+          showComponent={() =>
+            openWorkspaceTab(MedicationOrderBasket, "Medication Order")
+          }
+        >
+          <SummaryCardRow>
+            <SummaryCardRowContent>
+              <HorizontalLabelValue
+                label="Active Medications"
+                labelStyles={{
+                  color: "var(--omrs-color-ink-medium-contrast)",
+                  fontFamily: "Work Sans"
+                }}
+                value=" "
+                valueStyles={{
+                  color: "var(--omrs-color-ink-medium-contrast)",
+                  fontFamily: "Work Sans"
+                }}
+              />
+            </SummaryCardRowContent>
+          </SummaryCardRow>
+          <table className={styles.medicationsTable}>
+            <tbody>
+              {patientMedications.map((medication, index) => (
+                <tr key={index}>
+                  <td>
+                    <span
+                      style={{
+                        fontWeight: 500,
+                        color: "var(--omrs-color-ink-high-contrast)"
+                      }}
+                    >
+                      {medication.drug.name}
+                    </span>{" "}
+                    &mdash;{" "}
+                    <span
+                      style={{ color: "var(--omrs-color-ink-medium-contrast)" }}
+                    >
+                      {" "}
+                      DOSE
+                    </span>{" "}
+                    <span
+                      style={{
+                        fontWeight: 500,
+                        color: "var(--omrs-color-ink-high-contrast)"
+                      }}
+                    >
+                      {getDosage(
+                        medication?.drug?.strength,
+                        medication?.dose
+                      ).toLowerCase()}
+                    </span>{" "}
+                    &mdash;
+                    <span>
+                      {" "}
+                      {(medication?.doseUnits?.display).toLowerCase()}
```

Comment:
I would remove these parentheses.

Note: Its a difference if you write

```ts
(medication?.doseUnits?.display).toLowerCase()
medication?.doseUnits?.display.toLowerCase()
```

the former always calls `toLowerCase` (and thus runs it potentially on undefined) while the latter has it in the chain, i..e, it only runs it if the conditionals are fulfilled.

Reply:
Excellent